composer.json

```diff
@@ -9,14 +9,14 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Matomo\\Tests\\Ini\\": "tests/"
+            "Matomo\\Ini\\Tests\\": "tests/"
         }
     },
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=7.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.36",
-        "athletic/athletic": "0.1.*"
+        "phpunit/phpunit": "^7.5.20",
```

Comment:
Can't we directly switch to PHPUnit 8? According to https://phpunit.de/supported-versions.html it should support PHP 7.2

Reply:
Of course, fixed in 3dec719449af36a87856d9c9ae1ef82f99b3f242.